docker-compose.yml

```
@@ -0,0 +1,55 @@
+    restart: unless-stopped
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U ${POSTGRES_USER} -d ${POSTGRES_DB}"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    deploy:
+    resources:
+      limits:
+        memory: 1G
+    ports:
+    - "${POSTGRES_PORT:-5432}:5432"
```

yamllint (1.29.0-1)

[warning] wrong indentation: expected 6 but found 4

(indentation)